test/deploy/rackhd_source_install.py

```diff
@@ -108,7 +94,7 @@ def test02_clone_rackhd_source(self):
                                                  + fit_common.fitinstall()['rackhd']['branch']
                                                  )['exitcode'], 0, "Branch not found on RackHD repo.")
         # clone modules
-        for repo in modules:
+        for repo in fit_common.fitinstall().keys():
```

Comment:
As far as I can tell, cloning these repos is unnecessary. That gets handled in the playbook. I commented out this code and it worked fine.